roles/redhat_subscription/tasks/main.yml

```diff
@@ -21,14 +21,21 @@
 # Below we use string formatting to use a variable as the name of the file in
 # the lookup()
 # via http://stackoverflow.com/a/34239020
-#
-  - name: Get origin file
-    command: ostree admin --print-current-dir
-    register: origin_file
 
-  - name: Get refspec from origin file
-    command: grep refspec {{ origin_file.stdout }}.origin
-    register: refspec
+  - name: Determine is_atomic True/False
+    include: '{{ playbook_dir }}/../common/set_is_atomic.yml'
```

Comment:
Still failing:

```
TASK [atomic_host_check : Determine is_atomic True/False] **********************
fatal: [testnode]: FAILED! => {
    "failed": true, 
    "reason": "the file_name '/var/tmp/checkout/tests/common/set_is_atomic.yml' does not exist, or is not readable"
}
```

Try `include: ../../common/set_is_atomic.yml`

It's not the best solution, but would work in the short-term